client/app/bundles/shared/components/Footer.scss

```diff
@@ -0,0 +1,75 @@
+@import 'base';
+
+.footer {
```

Comment:
@nshki out of curiosity, is there a responsive mobile/tablet version of the footer?

Reply:
There should be, but I ended up not mocking one up, which is my bad! However, I think the columns should simply stack and text align center on smaller screens.

Reply:
Would be helpful to have mockups for that if you have any time :P